node/src/chain/acceptor.rs

```diff
@@ -270,6 +298,7 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
             fsv_bitset,
             ssv_bitset,
             block_time,
+            generator = blk.header().generator_bls_pubkey.to_base58(),
```

Comment:
As this is printed in Info level, why don't we use the truncated bs58 version?
```
{"level":"INFO","event":"block accepted","height":6,"iter":2,"hash":"ab91718e2b839959...f9e24b5b90e49eac","txs":0,"state_hash":"e72889f7b945edbf...3185aaf382334050","fsv_bitset":15,"ssv_bitset":14,"block_time":13,"generator":"rm6jwcM9xQChnVeHYw6W8HP7kW3E8nejgs4XhmKoYspQnrZL5dDcQjVR5e2UmbyNNF4DJDRdZGdYLgJfZ3qQXuhSCL6UTDkcQ2QEf6RW3aCtfqQMKCFMNaBsfNrZuzrP23Z","dur_ms":"5","target":"node::chain::acceptor"}

```

Reply:
For different reason:
- Previous node was logging it as full bs58 to properly match the staking address
- There is no truncated bs58 for the PublicKeyBytes

Anyway, we can get rid of previous impl, moving it to shortened bs58

Reply:
There is no value of having a full bs58 here. We may introduce (if not available already) an grpc/graphql API that prints full bs58 of the generator only on demand instead of polluting INFO level logs with extra data.